docs/features/profiles/marriages/divorcing.md

```diff
@@ -2,19 +2,11 @@
 
 ## How to divorce a user
 
-There are two ways to divorce a user.
+For your Information: This doesn't notify the person of your divorce.
 
-### 1. Divorce while mentioning
-
-Use the `-divorce @mention` divorce someone you're married to in a mutual server. This will ping the member you're divorcing.
-
-### 2. Divorce without mentioning
-
-This method doesn't notify the person of your divorce.
-
-1. Use the `-divorce` command to see a list of the people you're married to.
-2. From the list, pick the number of the person you want to divorce, then do `-divorce number` for example, if I want to divorce the second person on my list, I would use `-divorce 2`.
+1. Use the command `/divorce`.
+2. Click the dropdown menu that appears on the embed, click on the spouse that you want to divorce.
 
 {% hint style="info" %}
-This second method is especially useful if the person you want to divorce no longer shares any mutual servers with you or has deleted their account.
```

Comment:
It's important to mention this is how you divorce someone who isn't in the server. People ask this all the time